packages/vuetify/package.json

```diff
@@ -65,7 +65,7 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.5.5",
-    "@babel/core": "^7.5.5",
+    "@babel/core": "^7.0.0-0",
```

Comment:
Unsure about this downgrade too

Reply:
I'll double check the warnings associated.